### PR TITLE
Jedox OData Hub Connector

### PR DESCRIPTION
**Pull request by NGromann**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

### CODE REVIEW

File: certified-connectors/Jedox OData Hub/readme.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+
+## Jedox OData Hub Connector
+The Jedox OData Hub exposes functionality from different Jedox components into a powerful REST Api using the OData V4 standard.
+
```
Comment (sriyen-msft) [inline certified-connectors/Jedox OData Hub/readme.md:4]: @NGromann The publisher section is required. Please find a template for `readme.md` [here](https://github.com/microsoft/PowerPlatformConnectors/tree/dev/templates)

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/Jedox OData Hub/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,3804 @@
...
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/Databases": {
+      "get": {
+        "produces": [
+          "application/json; odata.metadata=minimal; odata.streaming=true",
+          "application/json"
```
[the remark this answers is not in the document]
Comment (NGromann) [reply certified-connectors/Jedox OData Hub/apiDefinition.swagger.json:23]: @sriyen-msft 
"application/json; odata.metadata=minimal; odata.streaming=true"
This one is used by all 200 responses.

"application/json"
This one is used for errors.

If you prefer removing those, I can do that. However I would then need to remove the non-200 responses aswell.

File: certified-connectors/Jedox OData Hub/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,3804 @@
...
+      "name": "Transforms"
+    },
+    {
+      "name": "Loads"
+    },
+    {
+      "name": "Jobs"
+    }
+  ],
+  "x-components": {},
```
[the remark this answers is not in the document]
Comment (NGromann) [reply certified-connectors/Jedox OData Hub/apiDefinition.swagger.json:3783]: @sriyen-msft  This property is a leftover from the tool we are using to downgrade the swagger version from 3 to 2. I would need to manually remove it on each update which might be error prone. If it is OK, I would prefer to keep this property.

File: certified-connectors/Jedox OData Hub/apiDefinition.swagger.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (sriyen-msft) [reply certified-connectors/Jedox OData Hub/apiDefinition.swagger.json:23]: @mamurshe Can you please help validate it?

File: certified-connectors/Jedox OData Hub/apiDefinition.swagger.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (sriyen-msft) [reply certified-connectors/Jedox OData Hub/apiDefinition.swagger.json:3633]: @NGromann You can remove this property manually if it is not getting generated by paconn tool.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

**Comment by sriyen-msft** [comment]
@NGromann I'm just checking in to see how things are going with updating your connector artifacts as per the given feedback. Please feel free to reach out with any questions you may have, Thanks.

**Comment by NGromann** [comment]
@sriyen-msft I quickly removed the x-components as you requested. For the other open issue I will be waiting for feedback.

File: certified-connectors/Jedox OData Hub/apiDefinition.swagger.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (sriyen-msft) [reply certified-connectors/Jedox OData Hub/apiDefinition.swagger.json:23]: @NGromann Please remove the second value, i.e. `"application/json"`.

**Comment by NGromann** [comment]
Hi @sriyen-msft, kindly let us know if there is something else that we should change.

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**